prow/scripts/cluster-integration/kyma-gke-load-test.sh

```diff
@@ -0,0 +1,355 @@
+        if [ "${hpa_test_status}" != "Running" ]; then
+            if [ "${hpa_test_status}" == "Succeeded" ]; then
+                succeeded='true'
+            fi
+            break
+        fi
+        sleep 5
+        echo "HPA tests are in progress..."
+    done
+
+    if [ $succeeded == 'true' ]; then
+        echo "Load test successfully completed!"
+    else
+        kubectl describe pod load-test -n kyma-system
+        echo 'Load test failed!'
+        echo 'Please check logs by executing: kubectl logs -n kyma-system load-test'
+        exit 1
+    fi
+}
+
+function checkPodIsRunning {
+    local POD_CHECK_TIMEOUT=700
+    local time_spent=0
+    while true; do
+        pod_status=$(kubectl get po -n kyma-system load-test -ojsonpath="{.status.phase}")
+        if [ "$pod_status" == "Running" ]; then
+            echo "Pod: load-test is running!"
+            break
+        fi
+        sleep 2
+        time_spent=$(( time_spent + 2 ))
+        if [[ $time_spent -gt $POD_CHECK_TIMEOUT ]]; then
+            echo "Timed out! pod: load-test is not running!"
+            echo "Description:"
+            kubectl describe pod load-test -n kyma-system
+            exit 1
+        fi
+    done
+
+}
+
+function installLoadTest() {
+	LT_FOLDER=${KYMA_SOURCES_DIR}/tools/load-test
+	LT_FOLDER_CHART=${LT_FOLDER}'/deploy/chart/load-test'
+
+	echo "---------------------------------------"
+	echo "- Executing load tests..."
+	echo "---------------------------------------"
+
+	echo "Installing helm chart..."
+	helm install --set slackClientToken="${SLACK_CLIENT_TOKEN}" \
+				--set slackClientChannelId="${LOAD_TEST_SLACK_CLIENT_CHANNEL_ID}" \
+				--set loadTestExecutionTimeout="${LT_TIMEOUT}" \
+				--set reqsPerRoutine="${LT_REQS_PER_ROUTINE}" \
+				"${LT_FOLDER_CHART}" \
+				--namespace=kyma-system \
+				--name=load-test
+
+	checkPodIsRunning
+
+	waitUntilHPATestIsDone
+	
+}
+
+function cleanup() {
+    OLD_CLUSTERS=$(gcloud container clusters list --filter="name~^${CLUSTER_NAME}" --format json | jq '.[].name' | tr -d '"')
+    CLUSTERS_SIZE=$(echo "$OLD_CLUSTERS" | wc -l)
+    if [[ "$CLUSTERS_SIZE" -gt 0 ]]; then
+	    shout "Delete old cluster"
+	    date
+	    for CLUSTER in $OLD_CLUSTERS; do
+		    removeCluster "${CLUSTER}"
+	    done
+    fi
+
+}
+
+shout "Authenticate"
+date
+init
+
+DNS_DOMAIN="$(gcloud dns managed-zones describe "${CLOUDSDK_DNS_ZONE_NAME}" --format="value(dnsName)")"
+export DNS_DOMAIN
+
+shout "Cleanup"
+date
+cleanup
+
+shout "Create new cluster"
+date
+createCluster
+
+shout "Install tiller"
+date
+
+shout "Account is:"
+gcloud config get-value account
+
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value account)"
+"${KYMA_SCRIPTS_DIR}"/install-tiller.sh
+
+shout "Install kyma"
+date
+installKyma
+
+shout "Install load-test"
+date
+installLoadTest
+
+shout "Cleanup after load test"
```

Comment:
General question: have you tried to make kmya-gke-long-lasting.sh script configurable to also support your case? Because now we have a lot of code duplication which can be then difficult to maintain.

Reply:
@aszecowka I've refactored `kyma-gke-load-test.sh` and the long-lasting job(along with all other jobs e.g. end to end test, gke integration, etc) needs to be refactored as well, which should be done in a separate ticket. All these jobs needs to be following the same approach as they follow a similar pattern.